e2e/frontend/queryEditor.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, test } from '@grafana/plugin-e2e';
+
+test.describe('Query Editor', () => {
+  test('data query is successful when query is valid', async ({ page, panelEditPage, selectors }) => {
+    await panelEditPage.datasource.set('SurrealDB');
+    await panelEditPage.setVisualization('Table');
+    // await panelEditPage.timeRange.set({ from: '2023-01-01', to: '2024-02-01' });
+
+    await page.waitForFunction(() => (window as any).monaco);
```

Comment:
Is it possible to avoid the `any`?

Reply:
This was taken from [here](https://github.com/grafana/grafana-aurora-datasource/blob/c833a3a22248f092d070231dc73d380fe33e6e42/e2e/tests/query-editor.spec.ts#L8), and I agree with their choice - it's not worth the effort to avoid `any` within a test file.